tests/unit_tests/test_wave_06.py

```diff
@@ -201,9 +201,10 @@ def test_swap_best_by_category_no_match_is_false():
     )
 
     assert result == False
-    assert tai.inventory[0].get_category() != "Clothing"
-    assert tai.inventory[1].get_category() != "Clothing"
-    assert tai.inventory[2].get_category() != "Clothing"
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert [item_a, item_b, item_c] == tai.inventory
+    assert [item_d, item_e, item_f] == jesse.inventory
```

Comment:
👍 This is a thorough check for all of the inventory items.

Since we don't expect a swap to occur, we could assume that the items are in the same positions they were when we created each Vendor, allowing us to use a list comparison rather than `in` checks.